CactbotOverlay/VersionChecker.cs

```diff
@@ -75,7 +77,10 @@ class VersionChecker {
     }
 
     private Advanced_Combat_Tracker.ActPluginData GetFFXIVPluginData() {
-      return GetPluginData("FFXIV_ACT_Plugin.dll");
+      if (language_ == "ko")
+        return GetPluginData("FFXIV_ACT_Plugin_Korean.dll");
+      else
+        return GetPluginData("FFXIV_ACT_Plugin.dll");
```

Comment:
If we have `FFXIV_ACT_Plugin_Korean.dll` but the plugin is falsely set to another language this will fail. Should we catch this?

Reply:
Yes, I think the korean dll check should come first, and if that dll is being used, it should always use the korean memory offsets.

Reply:
I've pushed another build that does this.